python/README.md

```diff
@@ -1,38 +1,202 @@
 # Cucumber Expressions for Python
 
-[The main docs are here](https://github.com/cucumber/cucumber-expressions#readme).
+[![PyPI Version](https://badge.fury.io/py/cucumber-expressions.svg)](https://pypi.org/project/cucumber-expressions/)
+[![Python versions](https://img.shields.io/pypi/pyversions/cucumber-expressions.svg)](https://pypi.org/pypi/cucumber-expressions)
+![LICENSE](https://img.shields.io/badge/license-MIT-blue)
+[![test-python](https://github.com/cucumber/cucumber-expressions/actions/workflows/test-python.yml/badge.svg)](https://github.com/cucumber/cucumber-expressions/actions/workflows/test-python.yml)
 
-## Build system
+For detailed documentation on __Cucumber Expressions__, visit the [main docs](https://github.com/cucumber/cucumber-expressions#readme).
 
-This project uses [Poetry](https://python-poetry.org/) as its build system.
-In order to develop on this project, please install Poetry as per your system's instructions on the link above.
+## Installation
 
-## Tests
+__Cucumber Expressions__ is available via [PyPI](https://pypi.org/project/cucumber-expressions/):
 
-The test suite uses `pytest` as its testing Framework.
+```console
+pip install cucumber-expressions
+```
+
+## Usage
+
+Cucumber Expressions can be used in numerous contexts as a human-friendly substitution for Regular Expressions.
+
+```python
+>>> from cucumber_expressions.expression import CucumberExpression
+>>> from cucumber_expressions.parameter_type_registry import ParameterTypeRegistry
+>>> registry = ParameterTypeRegistry()
+>>> expression = CucumberExpression("I have {int} cucumbers in my belly", registry)
+>>> expression.regexp
+'^I have ((?:-?\\d+)|(?:\\d+)) cucumbers in my belly$'
+>>> match = expression.match("I have 42 cucumbers in my belly")
+>>> match
+[<cucumber_expressions.argument.Argument object at 0x10dcf7650>]
+>>> argument = match[0]
+>>> argument.value
+42
+```
+
+### Using Cucumber Expressions with Behave
```

Comment:
This information is likely to go out of date without us noticing. It would be better to link directly to the behave docs.

Reply:
Ultimately end users are not expected to ever see this package. I would expect Behave to wrap this API with its own.

So this documentation can be focused purely on using this API and this API alone.

Reply:
This is also true for the other examples btw. But I think those are linked from the docs.cucumber.io site. 

This isn't great either but at least that is within the same project.

Reply:
That's fair - removed from PR. There's a slight gap at present between capturing this info between Cucumber and Behave docs - but hopefully Behave will add official native support at some point.